spec/issues/range_parameter_delimited_by_dash_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe '#XXX nested entity given as string' do
```

Comment:
please can you adapt the description, e.g. https://github.com/ruby-grape/grape-swagger/blob/master/spec/issues/582_file_response_spec.rb#L3 and also rename the spec file itself,
starting with issue number, thanks